### ISSUE TITLE
[Form field] Tooltip in Custom Label is not keyboard accessible and is not announced by screen readers

### ISSUE THREAD

**Issue by a11ymiko** [sl-design-system/components#3180]
## Current behavior 

In [Form field Custom Label story](https://storybook.sanomalearning.design/?path=/story/form-form-field--custom-label) mouse user can expand tooltip by hovering on icon inside label. 
Sighted keyboard users cannot set focus on icon to display tooltip.
Screen reader users using arrow can set screen reader's focus on icon, but nothing is announced there, so they loose information presented inside tooltip.

<img width="292" height="183" alt="Image" src="https://github.com/user-attachments/assets/9671c751-3a9e-4c7b-a867-f84569c21903" />

## Suggested solution

Label isn't active element, so adding "tabindex='0'" to make it focusable might be confusing for some users. 

Maybe adding second, alternative tooltip that is expanded when user sets focus on input would be a solution. People that are used to expanding mouse still could use icon inside label, but people who need focusable element to trigger tooltip could just go to `input`.
Example of that solution: https://dequeuniversity.com/library/aria/tooltip

### 👤 Your name

Kacper

### 🧢 Your Product/Team

SLDS

**Comment by jpzwarte** [comment]
I'm closing this issue since i already changed the story to use the infotip as part of #2111. .
